netbox/dcim/api/serializers_/devices.py

```diff
@@ -122,6 +122,7 @@ def get_config_context(self, obj):
 class VirtualDeviceContextSerializer(NetBoxModelSerializer):
     url = serializers.HyperlinkedIdentityField(view_name='dcim-api:virtualdevicecontext-detail')
     device = DeviceSerializer(nested=True)
+    identifier = serializers.IntegerField(allow_null=True, max_value=32767, min_value=0, required=False, default=None)
```

Comment:
was this missing previously?

Reply:
I needed to define the field to set `default=None` on it.